spec/features/user_spec.rb

```diff
@@ -60,3 +50,10 @@
     expect(page).to have_content('Password')
   end
 end
+feature 'User can\'t see a list of users' do
+  let(:user){ create_logged_in_user }
+  scenario 'clicks user list' do
```

Comment:
They're not actually clicking, right? They should not even see the link.

Reply:
changed